test/e2e/apps/cronjob.go

```diff
@@ -268,13 +269,50 @@ var _ = SIGDescribe("CronJob", func() {
 		err = deleteCronJob(f.ClientSet, f.Namespace.Name, cronJob.Name)
 		framework.ExpectNoError(err, "Failed to remove the %s cronjob in namespace %s", cronJob.Name, f.Namespace.Name)
 	})
+
+	// cleanup of failed finished jobs, with limit of one failed job by failedJobsHistoryLimit
```

Comment:
This test seems to be very similar to the previous one. Is there a way to reduce the duplication while keeping it readable?

Reply:
Thanks for the comment. I am considering about the way.

Reply:
@mortent
I modified to add failedJobsHistoryLimit to the existing test instead of adding new test.
Also, I changed deleteCronJob(). Because original function didn't delete the related jobs and pods, so the modified test didn't work properly. I checked that it did not affect existing tests in my local.

How about this change?